pkg/server/server.go

```diff
@@ -304,7 +304,16 @@ func (s *Server) HTTPHandler() http.Handler {
 	helmHandlers := helmhandlerspkg.New(s.KubeAPIServerURL, s.K8sClient.Transport)
 	handle("/api/helm/template", authHandlerWithUser(helmHandlers.HandleHelmRenderManifests))
 	handle("/api/helm/releases", authHandlerWithUser(helmHandlers.HandleHelmList))
-	handle("/api/helm/release", authHandlerWithUser(helmHandlers.HandleHelmInstall))
+	handle("/api/helm/release", authHandlerWithUser(func(user *auth.User, w http.ResponseWriter, r *http.Request) {
+		if r.Method == http.MethodGet {
+			helmHandlers.HandleGetRelease(user, w, r)
+		} else if r.Method == http.MethodPost {
+			helmHandlers.HandleHelmInstall(user, w, r)
+		} else {
+			serverutils.SendResponse(w, http.StatusMethodNotAllowed, serverutils.ApiError{Err: "Unsupported method, supported methods are GET, POST"})
+			w.Header().Set("Allow", "GET, POST")
```

Comment:
I believe this needs to be set before you commit the response with `SendResponse`

Reply:
Done